common/models/application.js

```diff
@@ -108,7 +108,9 @@ module.exports = function(Application) {
    * @param {String} owner Owner's user ID.
    * @param {String} name  Name of the application
    * @param {Object} options  Other options
-   * @param {Function} callback  Callback function
+   * @callback {Function} callback  Callback function
+   * @param {Error} err
```

Comment:
@bajtos In my opinion, we should make it explicit. Reasons:

1. Helps to give more details about the error, if required.
2. When a human being looks at the source code, if the error param is missing, it will be interpreted as the callback returns nothing. Only `sdocs` will be able to add the missing info. We should avoid magic.